test/Common/TestHelpers.cs

```diff
@@ -483,8 +483,8 @@ public static Task DeleteTaskHubResources(string testName, bool enableExtendedSe
             if (testName.Equals("TimerCancellation", StringComparison.OrdinalIgnoreCase) ||
                 testName.Equals("TimerExpiration", StringComparison.OrdinalIgnoreCase))
             {
-                // It is assumed that the 4th log message is a timer message.
-                timeStamp = GetTimerTimestamp(logMessages[3].FormattedMessage);
+                // It is assumed that the 5th log message is a timer message.
+                timeStamp = GetTimerTimestamp(logMessages[4].FormattedMessage);
```

Comment:
I didn't catch why this suddenly became the 5th log (whereas before in was the 4th). Can you please clarify?

Reply:
That was left over from the previous change when I was adding a new log for the runtime status change. For now, please ignore the changes in the test directory. I want to get feedback on the new approach (i.e. adding a new RuntimeStatus field to the existing state change log) before changing all of the tests since it's time consuming. Let me know if it's easier if I revert the changes in the test directory for now.

Reply:
Thanks for the clarification. Now that a reviews have been requested again, do we have some context on why the position of this message changed?